.changeset/cyan-squids-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-tools/build': patch
+---
+
+adds `@babel/core` as a peerDependency.
```

Comment:
just making sure this change was discussed with @TheSonOfThomp

Reply:
yeah, he's the one who suggested making it a patch update